Model/Transaction.php

```diff
@@ -121,9 +121,9 @@ public function getChildTransactions()
      */
     public function isPoiPiaPaymentMethod()
     {
-        $sPaymentId = $this->getPaymentType();
-        $oPayment = PaymentMethodHelper::getPaymentById($sPaymentId);
-        return is_subclass_of($oPayment->getPaymentMethod(), BasePoiPiaPaymentMethod::class);
+        $oXml = simplexml_load_string($this->getResponseXML());
+        $sPaymentId = (string) $oXml->{'payment-methods'}->{'payment-method'}['name'];
+        return $sPaymentId === 'wiretransfer' ? true : false;
```

Comment:
Please **remove** `? true : false` as it should not be needed.

Reply:
👍